db/fixtures/ae_datastore/ManageIQ/System/CommonMethods/QuotaMethods.class/__methods__/requested.rb

```diff
@@ -27,68 +31,105 @@ def request_totals(template_totals, dialog_totals)
 end
 
 def collect_template_totals(prov_option)
-  @service ? collect_totals(service_prov_option_value(prov_option)) : collect_totals(vm_prov_option_value(prov_option))
+  @service ? collect_totals(service_prov_option(prov_option)) : collect_totals(vm_prov_option_value(prov_option))
 end
 
 def get_option_value(request, option)
   request.get_option(option).to_i
 end
 
-def service_prov_option_value(prov_option, options_array = [])
+def service_prov_option(prov_option, options_array = [])
   @service_template.service_resources.each do |child_service_resource|
     if @service_template.service_type == 'composite'
       composite_service_options_value(child_service_resource, prov_option, options_array)
     else
       next if @service_template.prov_type.starts_with?("generic")
-      options_value(prov_option, child_service_resource.resource, options_array)
+      service_prov_option_value(prov_option, child_service_resource.resource, options_array)
     end
   end
   options_array
 end
 
+def service_prov_option_value(prov_option, resource, options_array = [])
+  args_hash = {:prov_option   => prov_option,
+               :array         => options_array,
```

Comment:
@tinaafitz This should be more descriptive than just `array`.  Seems like it should at least be `options_array` based on the other key names here.